pkg/auth/inject.go

```diff
@@ -121,6 +122,12 @@ func (m DependencyMap) Provide(dependencyName string, r *http.Request) interface
 			panic(err)
 		}
 		return trail
+	case "ForgotPasswordEmailSender":
+		tConfig := config.GetTenantConfig(r)
+		return forgotpwdemail.NewDefaultSender(tConfig, mail.NewDialer(tConfig.SMTP))
+	case "TestForgotPasswordEmailSender":
```

Comment:
Will it change like PR #805's "WelcomeEmailSendTask" to support async sender?

Reply:
Hmm, the original python plugin sends it synchronously, so I think I will just keep it like this.